pkg/net/net.go

```diff
@@ -93,10 +105,18 @@ type TaggedUnmarshaler interface {
 type BroadcastChannel interface {
 	// Name returns the name of this broadcast channel.
 	Name() string
-	// Send function publishes a message m to the channel. Message m needs to
+	// Send function publishes a message to the channel. Message needs to
 	// conform to the marshalling interface. Message will be periodically
-	// retransmitted by the channel for the lifetime of the provided context.
-	Send(ctx context.Context, m TaggedMarshaler) error
+	// retransmitted by the channel for the lifetime of the provided context
+	// according to the default StandardRetransmissionStrategy. Retransmission
+	// strategy can be set through the `strategy` vararg. If the vararg is
+	// given more than one value, the first value is used as the valid
+	// strategy.
+	Send(
+		ctx context.Context,
+		message TaggedMarshaler,
+		strategy ...RetransmissionStrategy,
```

Comment:
I would call this field `retransmissionStrategy`, just `strategy` does not tell what it is about without looking at the type. Is it pub-sub strategy, routing strategy, or what? This comment applies to all files with `strategy ...RetransmissionStrategy` in the function signature.

```suggestion
		retransmissionStrategy ...RetransmissionStrategy,
```

Reply:
Addressed in https://github.com/keep-network/keep-core/pull/3392/commits/a1a9ec4f552bc11bb7ee10c1436d90076b42fa05